src/runtime/recipe/handle-connection.ts

```diff
@@ -231,4 +231,13 @@ export class HandleConnection {
 
     return result.join(' ');
   }
+
+  getDisconnectedSpecsByHandleType() {
```

Comment:
findSpecsForUnnamedHandles

And add a TODO that the logic is wrong :slightly_smiling_face: